packages/itwin/tree-widget/src/components/trees/models-tree/internal/ModelsTreeVisibilityHandler.ts

```diff
@@ -263,7 +263,7 @@ class ModelsTreeVisibilityHandlerImpl implements HierarchyVisibilityHandler {
   private getSubjectNodeVisibilityStatus(subjectIds: Id64Array): Observable<VisibilityStatus> {
     const result = defer(() => {
       if (!this._props.viewport.view.isSpatialView()) {
-        return of(createVisibilityStatus("disabled", "subject.nonSpatialView"));
+        return of(createVisibilityStatus("disabled", "modelsTree.subject.nonSpatialView"));
```

Comment:
Maybe you could add a wrapper to avoid changing all stringId like:
```ts
function createVisibilityStatus(status, key) {
  return createVisibilityStatusCommon(status, `modelsTree.`${key});
}
```

Reply:
I would avoid wrapping it as it introduces unnecessary complexion, `createVisibilityStatus` have multiple implementations returning either `NonPartialVisibilityStatus` or `VisibilityStatus`, I also prefer seeing as much of the translation string as posible.